### PR TITLE
simplify animated_material example

### PR DESCRIPTION
**Pull request by mockersf**
# Objective

- example `animated_material` is more complex that needed to show how to animate materials
- it makes CI crash because it uses too much memory

## Solution

- Simplify the example

### CODE REVIEW

**Comment by alice-i-cecile** [comment]
I'd like to unbreak CI, but we should make a stress-test version of this to deliberately expose the performance issues that motivated its creation.

**Comment by mockersf** [comment]
this was not a stress test. examples should focus on one thing only

**Comment by alice-i-cecile** [comment]
I agree. I'm fine to merge this as is. I'll make an issue to stress-testify this.

**Review by alice-i-cecile — APPROVED** [review]

File: examples/3d/animated_material.rs [hunk 1]
```diff
@@ -25,11 +29,12 @@ fn setup(mut commands: Commands, asset_server: Res<AssetServer>) {
...
     ));
 
-    let helmet = asset_server.load("models/FlightHelmet/FlightHelmet.gltf#Scene0");
-    for x in -2..3 {
-        for z in -2..3 {
-            commands.spawn(SceneBundle {
-                scene: helmet.clone(),
+    let cube = meshes.add(shape::Cube { size: 0.5 });
+    for x in -1..2 {
+        for z in -1..2 {
```
Comment (rodolphito) [inline examples/3d/animated_material.rs:34]: if the materials aren't unique there's no point in spawning more than one helmet

**Review by rodolphito — CHANGES_REQUESTED** [review]

**Comment by rodolphito** [comment]
I'll make a PR to add this back as a stress test

File: examples/3d/animated_material.rs (hunk 1 above)
Comment (rodolphito) [reply examples/3d/animated_material.rs:34]: just realized this isnt helmets anymore

**Review by rodolphito — APPROVED** [review]

**Pull request merged**